neems-api/migrations/2025-10-13-203257_add_schedule_commands/up.sql

```diff
@@ -0,0 +1,38 @@
+-- Create schedule_commands table for battery charging commands
+CREATE TABLE schedule_commands (
+    id INTEGER PRIMARY KEY NOT NULL,
+    site_id INTEGER NOT NULL,
+    type TEXT NOT NULL CHECK (type IN ('charge', 'discharge', 'trickle_charge')),
```

Comment:
flagging this just as a tradeoff moment: if we were on psql we could use an enum type